tests/integration/aps/test_aps_api_client.py

```diff
@@ -0,0 +1,53 @@
+import json
+import os
+
+import pytest
+from aps.aps_api_client import APSApiClient
+from aps.aps_params import APSParams
+
+
+@pytest.fixture(scope="session")
+def vcr_config():
+    return {
+        "ignore_localhost": True,
+        "decode_compressed_response": True,
+        "filter_headers": ("X-Amz-Date"),
+        "record_mode": "once",
+    }
```

Comment:
if you add this here is just for this file, I would suggest to have it globally in the `tests/integration/conftest.py` file :)